ci-operator/step-registry/rhcs/e2e/teardown/rhcs-e2e-teardown-ref.yaml

```diff
@@ -25,6 +25,8 @@ ref:
     default: ""
   - name: "RHCS_VERSION" #if setup a job with RHCS_SOURCE the source in all tf manifest will be updated
     default: ""
+  - name: "ENABLE_SHARED_VPC"
```

Comment:
Is tear down part covered the share vpc clean up?

Reply:
Yes, see changes in `DestroyRHCSClusterByProfile` https://github.com/terraform-redhat/terraform-provider-rhcs/pull/569/files#diff-af47971dcdc8b7bd6bff3593ed96629cb6ce60b778725013948b3e9c5e133d0aR664-R712